src/main.py

```diff
@@ -0,0 +1,88 @@
+""" Main FastAPI app """
+import logging
+import os
+from pydantic import BaseModel, Field
+
+from typing import List
+
+from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
+from bayesian import run_one_1d_bayesian_optimization
+
+logger = logging.getLogger(__name__)
+
+version = "0.0.1"
+description = """
+"""
+app = FastAPI(
+    title="Materiom Active Learning API",
+    version=version,
+    description=description,
+    contact={
+        "name": "Materiom",
+    },
+)
+
+origins = os.getenv("ORIGINS", "*").split(",")
```

Comment:
What is this for?

Reply:
https://fastapi.tiangolo.com/tutorial/cors/ - ill added in a comment